src/openms/source/FORMAT/ExperimentalDesignFile.cpp

```diff
@@ -352,8 +357,8 @@ namespace OpenMS
       {
         // skip empty lines (except in state RUN_CONTENT, where the sample table is read)
         const String line(s.trim());
-	
-        if (line.empty() && state != RUN_CONTENT)
+	      // also skip comment lines
+        if (line.hasPrefix("#") || (line.empty() && state != RUN_CONTENT))
```

Comment:
We only allow empty outside of run content -as it seams. Could skipping a comment line not work for any reason? E.g. if line counts are needed outside of run content.

Reply:
I think nothing depends on Line numbers here, so we should be good. You could double-check.